Components/Utils/ModelRenderer.cs

```diff
@@ -177,8 +179,8 @@ public HtmlString TransactionalFooter()
                 SubTheme = _model.CdtsEnvironment.SubTheme,
                 ShowFooter = false,
                 ContactLinks = _model.Builder.BuildContactLinks(),
-                PrivacyLink = _model.Builder.BuildSingleFooterLink(_model.PrivacyLink),
-                TermsLink = _model.Builder.BuildSingleFooterLink(_model.TermsConditionsLink)
+                PrivacyLink = string.IsNullOrEmpty(_model.PrivacyLink.Href) ? null : new FooterLinkContext { ShowFooter = false, FooterLink = _model.PrivacyLink },
+                TermsLink = string.IsNullOrEmpty(_model.TermsConditionsLink.Href) ? null : new FooterLinkContext { ShowFooter = false, FooterLink = _model.TermsConditionsLink },
 
```

Comment:
This is not directly related to a specific code change but:   Footer() and TransactionalFooter() are also used with GCIntranet theme, no matter its subtheme, with basic or application template.  Did you confirm that this change of "terms/privacy link is not an array when showFooter is true" also works for gcintranet?

Reply:
Ah and also, that it is ok with GCweb appFooter

Reply:
will and not should, perfect